README.md

```diff
@@ -78,9 +78,28 @@ account.addPlugin({
         // Alwasy return a Promise
         return Promise.resolve(endpoint);
     },
-    // Will be called after the data was received bu before it is resolved to the client user
+    onError(endpoint, error) {
+        if (error === 'fetch failed') {
+            /* Silently spoof cached response */
+            endpoint.response = ResponseCache.lookup(endponint),
+            return Promise.resolve(endpoint);
+        } else {
+            /* Log and rethrow */
+            console.log(error);
```

Comment:
Logged errors are usually not easy to read. Either use console.error or just throw, the browser will display a thrown error.

Reply:
👍